_commands/basics/comm.md

```diff
@@ -0,0 +1,45 @@
+---
+---
+
+comm
+---
+Compares two sorted files line by line and writes three columns to standard output.
+<!-- one line explanation would go here -->
+
+<!-- minimal example -->
+~~~ bash
```

Comment:
This example is too long to be displayed on the reference page. Can a shorter example be given here?

Reply:
Modified the example to be smaller